bookwyrm/forms/custom_form.py

```diff
@@ -34,3 +35,43 @@ def save(self, request, *args, **kwargs):
         """Save and check perms"""
         self.instance.raise_not_editable(request.user)
         return super().save(*args, **kwargs)
+    
+    # def clean(self):
+    #     self.cleaned_data = super().clean()
+
+    #     print("Clean")
+    #     print(self.errors)
```

Comment:
This is empty at this position.

Which surprises me after reading documentation...